.github/workflows/file-system.yml

```diff
@@ -62,16 +62,16 @@ jobs:
       - name: Run
         run: mise run test-spm
 
-  test_linux:
-    name: "Test on Linux"
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v4
-      - uses: jdx/mise-action@v2
-        with:
-          experimental: true
-      - name: Run
-        run: mise run test-linux
+  # test_linux:
```

Comment:
Should we remove this?

Reply:
I initially thought to keep it for future reference, but yeah, let's get rid of it